crypto/src/test/kotlin/web5/sdk/crypto/AwsKeyManagerTest.kt

```diff
@@ -0,0 +1,70 @@
+package web5.sdk.crypto
+
+import com.amazonaws.AmazonServiceException
+import com.amazonaws.auth.AWSStaticCredentialsProvider
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.services.kms.AWSKMSClient
+import com.nimbusds.jose.JWSAlgorithm
+import com.nimbusds.jose.jwk.KeyType
+import com.nimbusds.jose.jwk.KeyUse
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import kotlin.test.Ignore
+import kotlin.test.assertEquals
+
+class AwsKeyManagerTest {
+
+  val signingInput = "The Magic Words are Squeamish Ossifrage".toByteArray()
+  val awsKeyManager = AwsKeyManager()
```

Comment:
Does this mean that this unit test is making network calls? Curious how this works.

Reply:
It doesn't 😃 Fighting with gradle/junit to disable them in GHA now